readme.md

```diff
@@ -1,36 +1,49 @@
-FlatBuffers Version 1.2.0
-
-# Welcome to FlatBuffers!
-
-## Build Status
-
-- Travis: [![Build Status](https://travis-ci.org/google/flatbuffers.svg?branch=master)](https://travis-ci.org/google/flatbuffers)
-
-----
-
-FlatBuffers is a serialization library for games and other memory constrained
-apps. Go to our [landing page][] to browse our documentation.
-
-FlatBuffers allows you to directly access serialized data without
-unpacking/parsing it first, while still having great forwards/backwards
-compatibility. FlatBuffers can be built for many different systems (Android,
-Windows, OS X, Linux), see `docs/html/index.html`
-
-Discuss FlatBuffers with other developers and users on the
-[FlatBuffers Google Group][]. File issues on the [FlatBuffers Issues Tracker][]
-or post your questions to [stackoverflow.com][] using the [`flatbuffers` tag][].
-
-To contribute to this project, see [CONTRIBUTING][].
-
+![logo](http://google.github.io/flatbuffers/fpl_logo_small.png) FlatBuffers [![Build Status](https://travis-ci.org/google/flatbuffers.svg?branch=master)](https://travis-ci.org/google/flatbuffers) 
+===========
+**FlatBuffers** is an efficient cross platform serialization library for games and 
+other memory constrained apps. It allows you to directly access serialized data without 
+unpacking/parsing it first, while still having great forwards/backwards compatibility. 
+
+**Go to our [landing page][] to browse our documentation.**
+
+## Supported operating systems
+* Android
+* Windows
+* MacOS X
+* Linux
+ 
+## Supported programming languages
+* C++
+* C#
+* Go
+* Java
+* JavaScript
+* PHP
+* Python
+* C (in progress)
```

Comment:
Instead of these two in-progress languages, maybe say `And more in progress` generically.